frappe/core/doctype/user_type/user_type.py

```diff
@@ -184,19 +189,14 @@ def add_role_permissions(doctype, role):
 	return name
 
 
-def get_non_standard_user_type_details():
+def get_non_standard_user_types():
```

Comment:
function is fairly internal, but happy to rename it back.

Reply:
eh https://sourcegraph.com/search?q=context%3Aglobal+get_non_standard_user_type_details+repo%3A%5Egithub%5C.com%2Ffrappe%2F.*&patternType=standard&groupBy=repo